dist/com.realm667.Wolfenstein_Blade_of_Agony.desktop

```diff
@@ -3,7 +3,8 @@ Name=Wolfenstein: Blade of Agony
 Comment=Story-driven FPS inspired by WWII shooters from the 90's and early 2000's
 GenericName=First Person Shooter Game
 Type=Application
-Icon=m_doom
+Icon=com.realm667.Wolfenstein_Blade_of_Agony
 Keywords=first;person;shooter;doom;
 Categories=Game;ActionGame;
-Exec=gzdoom
+Exec=gzdoom -iwad wolf_boa.ipk3 +fluid_patchset /usr/share/sounds/sf2/gzdoom.sf2 -file lights.pk3 brightmaps.pk3 game_widescreen_gfx.pk3
```

Comment:
I just saw this. AFAIK GZDoom needs Fluidsynth 1.x to work properly, and distributions like Ubuntu 20.04 use Fluidsynth 2.x, so maybe you should set LD_LIBRARY_PATH on this line as well? Maybe it won't be an issue, since Blade of Agony doesn't use any MIDI tracks.

Reply:
You're right. I ran Wolfenstein with gzdoom 4.5.0 and fluidsynth 2, and it doesn't even need the '+fluid_patchset /usr/share/sounds/sf2/gzdoom.sf2' option.